daml-lf/parser/BUILD.bazel

```diff
@@ -21,6 +21,7 @@ da_scala_library(
     visibility = [
         "//daml-lf:__subpackages__",
         "//ledger:__subpackages__",
+        "//triggers:__subpackages__",
```

Comment:
Trigger simulations are using the LF parser to simplify defining initial starting states for trigger simulations